src/thermal_conductivity/options.f90

```diff
@@ -200,6 +181,23 @@ subroutine parse(opts)
     ! Get things to atomic units
     opts%mfp_max = opts%mfp_max*lo_m_to_Bohr
 
+    if (maxval(opts%qg4ph) .gt. 0 .and. .not. opts%fourthorder) then
+        write(*, *) 'You have to enable fourthorder to use a fourth order Monte-Carlo grid, stopping calculation.'
+        stop
+    end if
+
+    ! Set automatic values for Monte-Carlo grids
+    if (opts%thirdorder) then
```

Comment:
I like the idea that by default, you have the full grids. For example, this could still be used for cases where third order is not really expensive but where you just use Monte-Carlo for the fourth order

Reply:
ok good